src/main/java/com/sap/sgs/phosphor/fosstars/tool/format/PrettyPrinter.java

```diff
@@ -46,42 +46,18 @@
   private static final Map<Class, String> FEATURE_CLASS_TO_NAME = new HashMap<>();
 
   static {
-    FEATURE_CLASS_TO_NAME.put(
-        OssSecurityScore.class,
-        "Security of project");
-    FEATURE_CLASS_TO_NAME.put(
-        CommunityCommitmentScore.class,
-        "Community commitment");
-    FEATURE_CLASS_TO_NAME.put(
-        ProjectActivityScore.class,
-        "Project activity");
-    FEATURE_CLASS_TO_NAME.put(
-        ProjectPopularityScore.class,
-        "Project popularity");
-    FEATURE_CLASS_TO_NAME.put(
-        ProjectSecurityAwarenessScore.class,
-        "Security awareness");
-    FEATURE_CLASS_TO_NAME.put(
-        ProjectSecurityTestingScore.class,
-        "Security testing");
-    FEATURE_CLASS_TO_NAME.put(
-        UnpatchedVulnerabilitiesScore.class,
-        "Unpatched vulnerabilities");
-    FEATURE_CLASS_TO_NAME.put(
-        VulnerabilityLifetimeScore.class,
-        "Vulnerability lifetime");
-    FEATURE_CLASS_TO_NAME.put(
-        MemorySafetyTestingScore.class,
-        "Memory-safety testing");
-    FEATURE_CLASS_TO_NAME.put(
-        UsesFindSecBugs.class,
-        "Uses FindSecBugs");
-    FEATURE_CLASS_TO_NAME.put(
-        UsesNoHttpTool.class,
-        "Uses nohttp");
-    FEATURE_CLASS_TO_NAME.put(
-        DependencyScanScore.class,
-        "Dependency testing");
+    FEATURE_CLASS_TO_NAME.put(OssSecurityScore.class, "Security of project");
```

Comment:
Is this format alright? I liked the earlier one. It is just my preference to be readable.

Reply:
Yes, this just updates formatting. The number of features and score are going to grow. Let's make the code a bit more compact.